src/hooks/useStore.ts

```diff
@@ -0,0 +1,56 @@
+import { create } from 'zustand'
```

Comment:
We should move these hooks over a `store` folder and perhaps use a separate file for each one.

Reply:
i.e. `src/store/useConfigStore.ts` and `src/store/usePaginationStore.ts`